INSTALL.md

```diff
@@ -1,23 +1,114 @@
+  * [Specific Application Dependencies](#specific-application-dependencies)
+* [Basic Configuration](#basic-configuration)
+* [Adding Applications](#adding-applications)
+* [Adding Kratos to Path](#adding-kratos-to-path)
+* [Examples](#examples)
+  * [Linux](#linux)
+  * [Windows](#windows)
+  * [MacOS](#macos)
+* [Advanced Configuration](#advanced-configuration)
+  * [Building Environment](#building-environments)
+  * [Common Flags](#common-flags)
+  * [Compilation Performance](#compilation-performance)
+  * [Parallelism](#parallelism)
+  * [External Libraries](#external-libraries)
+    * [Feast](#feast)
+    * [Metis](#metis)
+    * [Trilinos](#trilinos)
+
+## Cloning Kratos
+
+In order to obtain the source code of Kratos, you will need to clone the repository using git.
+
+You can install git through the following command in Linux:
+```Shell
+sudo apt-get install git
+```
+In Windows, you can download it in:
+
+* [Download Git](https://git-scm.com/downloads)
+
+
+
+Once git is installed you can fetch the code by using this command in a terminal:
+
+```Shell
+git clone https://github.com/KratosMultiphysics/Kratos Kratos
+```
+
+## Kratos Dependencies
+
+### Kratos Core Dependencies
+  This are the basic dependecies needed to compile the Kratos Core and most of the Kratos applications.
+  * Python3-dev
+  * C++11 compiler
+  * CMake
+  * Boost
+
+Additionaly Visual Studio or MinGW is required to compile in Windows.
+
+- #### Linux installation
+
+    The command below will install all the packages needed.
+
+    ```Shell
+    sudo apt-get install python3-dev gcc g++ cmake libboost-all-dev
+    ```
+    Newer versions of boost can be downloaded in:
+
+    http://www.boost.org/users/download/.
+
+- #### Windows installation
+
+    - Visual Studio
+
+        *Visual Studio* is the only compiler officially supported to build *Kratos* under *Windows*.
+        Since the adoption of **C++11** we support versions 2015 update 3 onwards. We recommend you to use the latest version of visual studio which can be obtained here for free:
+
+        * [Download Visual Studio](https://visualstudio.microsoft.com/en/thank-you-downloading-visual-studio/?sku=Community&rel=15)
+
+        Since *Visual Studio* is a multi-language IDE, some distributions come without C++ compiler. Please, make sure that you can create a C++ project before continue, in case C++ packages were missing you will be prompt to download them.
+
+    - CMake
+        * [Download CMake](http://cmake.org/download/)
+
+        Once installing, please <span style="color:red"> do not forget to mark the option: '''"Add CMake to the system PATH for all users"'''</span>
+
+        Please notice that if you want to use *python* 3.4 or higher, you will need *CMake* 3.0.2 or higher.
+
+    - Python
+
+        You will need any version of python in your computer in order to compile *Kratos*. We strongly recommend *Python* 3, at least 3.3.4 or higher. you can download python from its official webpage:
```

Comment:
@roigcarlo  What about  the minimum python 3.3.4? Is it up to date?

Reply:
Better Python 3.5 at least, with 3.7 / 3.8 recommended